src/tox_uv/_venv.py

```diff
@@ -41,6 +42,7 @@
     "system",
     "only-system",
 ]
+logger = logging.getLogger(__name__)
```

Comment:
```python
_LOGGER: Final[logging.Logger] = logging.getLogger(__name__)
```

Reply:
If would change this separate to keep it consistent with other areas.